src/containers/Projects/ProjectDetails/ProjectDetailsSideMenu/index.tsx

```diff
@@ -82,12 +82,14 @@ const ProjectDetailsSideMenu: FC = () => {
             spy
             to={anchor}
           >
-            <ProjectIcon
-              className="ProjectDetailsSideMenu__icon"
-              icon={iconType}
-              size={shouldShowDetails ? ProjectIconSize.medium : ProjectIconSize.small}
-              state={iconState}
-            />
+            {iconType && (
```

Comment:
@jamessspanggg this is a hack to support current project details page, as I have made `iconType` optional.

Reply:
ok noted